worlds/oot_soh/location_access/overworld/kokiri_forest.py

```diff
@@ -217,30 +244,6 @@ def set_region_rules(world: "SohWorld") -> None:
         (Regions.KOKIRI_FOREST, lambda bundle: True)
     ])
 
-    ## KF Outside Deku Tree
-    # Locations
-    add_events(Regions.KF_OUTSIDE_DEKU_TREE, world, [
-        (EventLocations.MIDO_OUTSIDE, LocalEvents.MIDO_SWORD_AND_SHIELD, lambda bundle: has_item(Items.KOKIRI_SWORD, bundle) and
-                                                  has_item(Items.DEKU_SHIELD, bundle)),
-    ])
-    add_locations(Regions.KF_OUTSIDE_DEKU_TREE, world, [
-        (Locations.KF_DEKU_TREE_LEFT_GOSSIP_STONE_FAIRY, lambda bundle: call_gossip_fairy_except_suns(bundle)),
-        (Locations.KF_DEKU_TREE_LEFT_GOSSIP_STONE_BIG_FAIRY, lambda bundle: can_use(Items.SONG_OF_STORMS, bundle)),
-        (Locations.KF_DEKU_TREE_RIGHT_GOSSIP_STONE_FAIRY, lambda bundle: call_gossip_fairy_except_suns(bundle)),
-        (Locations.KF_DEKU_TREE_RIGHT_GOSSIP_STONE_BIG_FAIRY, lambda bundle: can_use(Items.SONG_OF_STORMS, bundle))
-    ])
-    # Connections
-    connect_regions(Regions.KF_OUTSIDE_DEKU_TREE, world, [
-        (Regions.DEKU_TREE_ENTRYWAY, lambda bundle: (is_child(bundle))
-                                                         and (world.options.closed_forest.value == 2
-                                                            or has_item(LocalEvents.MIDO_SWORD_AND_SHIELD, bundle))),
-        (Regions.KOKIRI_FOREST, lambda bundle:  (is_adult(bundle) and
-                                                             (can_pass_enemy(bundle, Enemies.BIG_SKULLTULA) or
-                                                             has_item(Events.CLEARED_FOREST_TEMPLE, bundle)))
-                                                            or has_item(LocalEvents.MIDO_SWORD_AND_SHIELD, bundle)
-                                                           or world.options.closed_forest.value == 2)
-    ])
-
     ## KF Storms Grotto
     # Locations
     add_locations(Regions.KF_STORMS_GROTTO, world, [
```

Comment:
This is missing all the grotto events. This is what's on the ship side: 
<img width="659" height="113" alt="image" src="https://github.com/user-attachments/assets/97316158-a314-4321-945f-633107cfebc5" />

I *may* turn this into a helper of some kind too, but even then it'd be nice to see it added here so I can turn it into a helper after.

Reply:
Never mind this needs unique EventLocations per grotto on the AP side so there won't be a helper either way.